src/librustc/dep_graph/dep_tracking_map.rs

```diff
@@ -55,7 +55,7 @@ impl<M: DepTrackingMapConfig> MemoizationMap for RefCell<DepTrackingMap<M>> {
     ///
     /// ```
     /// fn type_of_item(..., item: &hir::Item) -> Ty<'tcx> {
-    ///     let item_def_id = ccx.tcx.hir().local_def_id(it.id);
+    ///     let item_def_id = ccx.tcx.hir().local_def_id_from_node_id(it.id);
```

Comment:
This should be `local_def_id` now,

Reply:
Good catch; also, the `id` was changed to `hir_id`.